__init__.py

```diff
@@ -32,6 +33,8 @@ def icon():
     return "icon.png"
 def qgisMinimumVersion():
     return "1.7"
+"""
```

Comment:
This quotes should be removed from here. You are commenting a block of functional code.